.github/workflows/check-for-non-releasable-actions.yaml

```diff
@@ -0,0 +1,71 @@
+name: Check for non-releasable actions
+on:
+  pull_request:
+    # paths:
+    #   - actions/
+    types:
+      - edited
+      - opened
+      - ready_for_review
+      - synchronize
+
+  push:
+    branches:
+      - main
+    paths:
+      - actions/
+
+jobs:
+  check-for-non-releasable-actions:
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          sparse-checkout: |
+            ./actions
+            ./release-please-config.json
+
+      - name: List folders
+        id: list-folders
+        uses: Drafteame/list-folders-action@2138be65c1ec39ea5357a0d47263f330d28b4b02 # v1.0.0
+        with:
+          paths: |
+            ./actions
+
+      - name: Install jsonnet
+        run: go install github.com/google/go-jsonnet/cmd/jsonnet@latest
```

Comment:
Please cache the jsonnet binary so that this doesn't have to be run every time 🙂

Reply:
@zerok https://github.com/grafana/shared-workflows/actions/runs/12082633004/job/33694117294